target/rspamd/local.d/worker-controller.inc

```diff
@@ -0,0 +1,3 @@
+# documentation: https://rspamd.com/doc/workers/controller.html
+
+bind_socket = "0.0.0.0:11334";
```

Comment:
Is this one meant to be `0.0.0.0` or `127.0.0.1` like the others?

Reply:
It should `0.0.0.0` here and `127.0.0.1` for the others. The controller exposes a web interface, and I want to make sure I can expose this, for example via `Traefik`. So I think we need `0.0.0.0` for the controller. Please correct me if I'm wrong :)

Reply:
To be reachable outside of the container (_that includes another container performing as a reverse-proxy_), you have to bind to a common network AFAIK, so that should be fine.

That wouldn't require binding the port on the Docker host public networks either. I just wanted to verify the difference :+1: